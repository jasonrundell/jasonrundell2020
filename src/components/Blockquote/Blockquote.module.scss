.root {
  position: relative;
  &::before {
    display: inline;
    position: absolute;
    font-family: Times New Roman, Times, serif;
    font-size: 3rem;
    color: #9aaec6;
    opacity: 0.8;
    content: '❝';
    top: -1.5rem;
    left: -2.25rem;
  }
  &::after {
    display: inline;
    position: absolute;
    font-family: Times New Roman, Times, serif;
    font-size: 3rem;
    color: #9aaec6;
    opacity: 0.8;
    content: '❞';
    right: -1.5rem;
  }
}
