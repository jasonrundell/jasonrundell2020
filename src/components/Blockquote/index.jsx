import React from 'react'
import PropTypes from 'prop-types'

// CSS
import styles from './Blockquote.module.scss'

const Blockquote = ({ children }) => {
  return <blockquote className={styles.root}>{children}</blockquote>
}

Blockquote.propTypes = {
  children: PropTypes.any.isRequired,
}

export default Blockquote
