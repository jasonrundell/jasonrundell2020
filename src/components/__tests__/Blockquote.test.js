import React from 'react'
import { create } from 'react-test-renderer'

import Blockquote from '../Blockquote'

describe('Blockquote component', () => {
  test('Matches the snapshot', () => {
    const component = create(<Blockquote>Blockquote child</Blockquote>)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
