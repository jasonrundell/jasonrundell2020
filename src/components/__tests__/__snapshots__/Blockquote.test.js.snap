// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blockquote component Blockquote renders correctly 1`] = `
<blockquote
  className="root"
>
  Blockquote component
</blockquote>
`;
